src/Stratis.Bitcoin.Features.Consensus/Rules/CommonRules/PosCoinviewRule.cs

```diff
@@ -98,7 +96,7 @@ protected override void UpdateCoinView(RuleContext context, Transaction transact
         }
 
         /// <inheritdoc />
-        protected override void CheckMaturity(UnspentOutputs coins, int spendHeight)
+        public override void OnCheckMaturity(UnspentOutputs coins, int spendHeight)
```

Comment:
I wonder why add the `On` keyword, On would indicate to me that an even happened (like OnClick)

Reply:
I have now removed these :+1